internal/helpers.go

```diff
@@ -560,16 +561,16 @@ func ResetK1Dir(k1Dir string) error {
 		}
 	}
 
-	if _, err := os.Stat(k1Dir + "/gitops"); !os.IsNotExist(err) {
-		err := os.RemoveAll(k1Dir + "/gitops")
+	if _, err := os.Stat(k1Dir + "/" + gitopsRepoName); !os.IsNotExist(err) {
+		err := os.RemoveAll(k1Dir + "/" + gitopsRepoName)
 		if err != nil {
-			return fmt.Errorf("unable to delete %q folder, error: %s", k1Dir+"/gitops", err)
+			return fmt.Errorf("unable to delete %q folder, error: %s", k1Dir+"/"+gitopsRepoName, err)
```

Comment:
I know this was copied from the old code and just part of the line changed, but this is a good moment to start replacing those `%s` error handles with `%w` (which allows for unit-testing and comparison of errors):

```suggestion
			return fmt.Errorf("unable to delete %q folder, error: %w", k1Dir+"/"+gitopsRepoName, err)
```